src/hbbft_acs.erl

```diff
@@ -308,12 +308,16 @@ deser_bba_key(AK) ->
     "bba_" ++ Int = atom_to_list(AK),
     list_to_integer(Int).
 
--spec serialize_rbc_state(rbc_state()) -> binary().
+-spec serialize_rbc_state(rbc_state()) -> map().
+serialize_rbc_state(#rbc_state{rbc_data=RBCData, result=undefined}) ->
+    #{data => hbbft_rbc:serialize(RBCData)};
 serialize_rbc_state(#rbc_state{rbc_data=RBCData, result=Result}) ->
-    term_to_binary(#rbc_serialized_state{rbc_data=RBCData, result=Result},
-                   [compressed]).
+    #{result => Result, data => hbbft_rbc:serialize(RBCData)}.
 
 -spec deserialize_rbc_state(rbc_serialized_state() | #{}) -> rbc_state().
+deserialize_rbc_state(#{ data := RBCData}=Data) ->
+    %% new RBC sub serialization
+    #rbc_state{rbc_data=RBCData, result=maps:get(result, Data, undefined)};
```

Comment:
if these are all accepting the ``#rbc_serialized_state{}` why do we need to call `hbbft_rbc:serialize/1` going in but we aren't calling `hbbft_rbc:deserialize/1` coming back out from the rbc_serialized_state to the rbc_state?